release/ray_release/scripts/ray_bisect.py

```diff
@@ -54,14 +56,27 @@
     default=False,
     help=("Use the full, non-smoke version of the test"),
 )
+@click.option(
+    "--global-config",
+    default="oss_config.yaml",
+    type=click.Choice(
+        [x.name for x in (Path(__file__).parent.parent / "configs").glob("*.yaml")]
+    ),
+    help="Global config to use for test execution.",
+)
 def main(
     test_name: str,
     passing_commit: str,
     failing_commit: str,
     concurrency: int = 1,
     run_per_commit: int = 1,
     is_full_test: bool = False,
+    global_config: str = "oss_config.yaml",
 ) -> None:
+    global_config_file = os.path.join(
+        os.path.dirname(__file__), "..", "configs", global_config
+    )
```

Comment:
should this be using bazel runfiles? or support both pattern?

Reply:
supporting both sounds good!